tools/oas-apigee-mock/lib/commands/generateApi/generateApi.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var parser = require('swagger-parser')
+var generateSkeleton = require('./generateSkeleton.js')
+var generateProxy = require('./generateProxy.js')
+var generatePolicies = require('./generatePolicies.js')
+var generateProxyEndPoint = require('./generateProxyEndPoint.js')
+var async = require('async')
+var path = require('path')
+
+module.exports = {
+  generateApi: generateApi
+}
+
+function generateApi(apiProxy, options, cb) {
```

Comment:
General consideration for the function. Use `async function` with `await` instead of callbacks to avoid deeply nested callbacks. This would improve readability.